DimaZ_NewZ/urls.py

```diff
@@ -16,7 +16,19 @@
 """
 from django.contrib import admin
 from django.urls import path
+from hello import views
+from django.conf import settings  # Import the settings to access MEDIA_URL and MEDIA_ROOT
+from django.conf.urls.static import static  # Import static to serve media files during development
+
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', views.home, name='home'),  # Route for the home page
+    path('news/', views.all_news, name='all_news'),  # Route for the full news list
```

Comment:
it would be best if you listed it in the news app. After that, you should include the news app url.py file here